starters/basic-starter/pages/index.tsx

```diff
@@ -88,6 +90,12 @@ export default function IndexPage({ menus, events, places }: IndexPageProps) {
 export async function getStaticProps(
   context,
 ): Promise<GetStaticPropsResult<IndexPageProps>> {
+  // Test server connection on development environment. This translates common
+  // API errors to more developer friendly error messages.
+  if (process.env.NODE_ENV === 'development') {
+    await testApiConnection(CONTENT_TYPES, drupal);
```

Comment:
If we decide to go with this solution, we should remove this change from this PR until we have released the code in a "next-acms" release.